app/bundles/PointBundle/Model/PointModel.php

```diff
@@ -54,12 +55,23 @@ class PointModel extends CommonFormModel
      */
     protected $mauticFactory;
 
-    public function __construct(Session $session, IpLookupHelper $ipLookupHelper, LeadModel $leadModel, MauticFactory $mauticFactory)
-    {
+    /**
+     * @var ContactTracker
+     */
+    protected $contactTracker;
```

Comment:
Any reason why not `private`?

Reply:
not specific one. The others properties are protected so I take them protecte.

Reply:
Could you please make the new ones private?